src/textual/widgets/_list_view.py

```diff
@@ -155,13 +162,16 @@ def clear(self) -> AwaitRemove:
         return await_remove
 
     def action_select_cursor(self) -> None:
+        """Select the current item in the list."""
         selected_child = self.highlighted_child
         self.post_message_no_wait(self.Selected(self, selected_child))
 
     def action_cursor_down(self) -> None:
+        """Select the next item in the list."""
         self.index += 1
 
     def action_cursor_up(self) -> None:
+        """Select the previous item in the list."""
```

Comment:
These docstrings are a little misleading. Pressing up/down doesn't select the previous/next items, it highlights them.

Reply:
Good catch, poor wording on my part.